template/.travis.yml

```diff
@@ -0,0 +1,80 @@
+sudo: false
+
+language: php
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+env:
+  global:
+    - COMPOSER_ARGS="--no-interaction"
+    - COVERAGE_DEPS="satooshi/php-coveralls"
+    - LEGACY_DEPS="phpunit/phpunit"
+
+matrix:
+  include:
+    - php: 5.6
```

Comment:
For new packages, we only need 7.1 and nightly; we can remove the 5.6 and 7.0 targets (though you'll have to move the `TEST_COVERAGE` env stuff to the 7.1 locked env).